toonz/sources/toonz/CMakeLists.txt

```diff
@@ -416,7 +419,12 @@ endif()
 qt5_wrap_cpp(SOURCES ${MOC_HEADERS})
 
 if(BUILD_ENV_MSVC)
-    add_executable(Tahoma2D WIN32 ${HEADERS} ${SOURCES} ${OBJCSOURCES} ${RESOURCES} ${RC_FILE})
+    add_executable(Tahoma2D WIN32 ${HEADERS} ${SOURCES} ${OBJCSOURCES} ${RESOURCES} ${RC_FILE}
+        vectorinspector.h
```

Comment:
Why are .h and .cpp files listed here?  They are already mentioned above.

Reply:
I'll resolve it.

Reply:
Completed.